src/components/TableDataView/TableDataView.jsx

```diff
@@ -4,7 +4,7 @@ import { Table } from 'semantic-ui-react';
 
 import './styles.less';
 
-const DataView = ({ tableData }) => {
```

Comment:
@dana-cfc4 Hm. I don't think renaming is needed. Can you please revert these changes?

Reply:
DataView comes as a global object in javascript. Thus, we can rename it to avoid sonarqube : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView